redux/auth/reducer.ts

```diff
@@ -47,6 +54,36 @@ function reducer(
 
     case LOGOUT:
       return initialState;
+    case SIGNUP:
+      return {
+        ...state,
+        requestStatus: RequestStatus.LOADING,
+      };
+
+    case SIGNUP_SUCCESS:
+      return {
+        ...state,
+        token: action.payload.token,
+        requestStatus: RequestStatus.SUCCESS,
+      };
+
+    case SIGNUP_FAILURE:
+      return {
+        ...state,
+        token: null,
+        requestStatus: RequestStatus.FAILURE,
+      };
+
+    case SET_USER_DATA: {
```

Comment:
Please remove this action and related code as stated in the comment in `redux/auth/saga.ts` file.